web/src/app/donate/PayWithPayPalComponent.js

```diff
@@ -1,81 +1,75 @@
 import paypal from 'paypal-checkout';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import RestClient from '../RestClient';
+
+const sandboxID = 'AV_VKTQQlMAB8Fq58souv8_CazzqdHZS5pDF_H9L4eZIIH65I6LjvoSQVBHmgCk9-uJwtpsKxfOX9bop';
```

Comment:
Is this something that is safe to put in code?

Reply:
absolutely not! But ill handle fixing that when i set up the beta paypal accounts. Lets leave this as is for now

Reply:
Is there a mechanism for scrubbing this from git history?